virenv/bin/jp.py

```diff
@@ -0,0 +1,63 @@
+#!/root/corio/virenv/bin/python
```

Comment:
We are skipping virenv directory and related files to be checked in in CORIO. Why this file is added?

Reply:
Yes @SCP-Seagate  we need to remove this virenv directory.